content/docs/services/intro.md

```diff
@@ -5,12 +5,36 @@ menu:
 aliases: 
     - /docs/services/
 title: Introduction
-layout: services
 weight: -1
 ---
 
-cloud.gov offers multiple [managed services]({{< relref "docs/apps/managed-services.md" >}}) to allow your application to expand its functionality. You can also create [user-provided services]({{< relref "docs/apps/production-ready.md#protect-access-to-sensitive-credentials" >}}).
+You can expand the functionality of your cloud.gov application by making use of services. Before your application can use a service, the service must be provisioned, and credentials for using the service must be supplied to your application. 
 
-To list all the managed services and plans available to your organization, you can run `cf marketplace` from your command line.
+There are two ways to provision services:
+
+- You can provision a service manually outside of cloud.gov, then create a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html) to hold the credentials.
+- You can provision a [managed service instance](https://docs.cloudfoundry.org/devguide/services/#instances) through the marketplace in cloud.gov on demand.
+
+In both cases, you make the credentials available by [binding the service instance to your application](https://docs.cloudfoundry.org/devguide/services/application-binding.html). 
+
+## Marketplace services
```

Comment:
Consider making this an action phrase ("Provision marketplace services") so readers know what value it offers them when skimming.